.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish to testpypi repo
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Run build script
+      run: >-
+        bash build_release.sh
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.pypi_password }}
```

Comment:
I assume we have to configure this in the main repo before merging this?

Reply:
I have already configured this for the main repo.

Reply:
And the secret is a generated API token? does it expire?